src/components/Alteration/Articles/CompanyProvisions.vue

```diff
@@ -1,26 +1,33 @@
 <template>
   <div id="company-provisions">
-    <v-container class="pa-0 ma-0" v-if="!isEditing">
+    <template v-if="!isEditing">
       <v-row no-gutters>
-        <v-col class="pr-0 pl-0 pt-0" cols="3">
```

Comment:
This is a misunderstanding of what `no-gutters` does. Not needed.

Ditto below where I removed 0 padding and margins.